[2024-01-30T15:29:34.526+0100] {processor.py:161} INFO - Started process (PID=6567) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:29:34.533+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:29:34.541+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:29:34.539+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:29:34.626+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:29:34.621+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:29:34.628+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:29:34.765+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.360 seconds
[2024-01-30T15:30:45.594+0100] {processor.py:161} INFO - Started process (PID=6639) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:30:45.599+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:30:45.600+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:30:45.600+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:30:45.605+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:30:45.604+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:30:45.606+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:30:45.643+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.053 seconds
[2024-01-30T15:31:16.483+0100] {processor.py:161} INFO - Started process (PID=6673) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:31:16.486+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:31:16.490+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:31:16.489+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:31:16.498+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:31:16.497+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:31:16.499+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:31:16.613+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.135 seconds
[2024-01-30T15:32:03.385+0100] {processor.py:161} INFO - Started process (PID=6725) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:32:03.389+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:32:03.390+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:32:03.389+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:32:03.394+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:32:03.393+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:32:03.394+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:32:03.536+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.155 seconds
[2024-01-30T15:32:41.678+0100] {processor.py:161} INFO - Started process (PID=6761) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:32:41.697+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:32:41.720+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:32:41.711+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:32:41.854+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:32:41.836+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:32:41.856+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:32:42.119+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.500 seconds
[2024-01-30T15:33:29.090+0100] {processor.py:161} INFO - Started process (PID=6808) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:33:29.092+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:33:29.094+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:33:29.093+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:33:29.098+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:33:29.097+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:33:29.098+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:33:29.126+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.039 seconds
[2024-01-30T15:34:00.713+0100] {processor.py:161} INFO - Started process (PID=6837) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:34:00.715+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:34:00.717+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:34:00.716+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:34:00.721+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:34:00.720+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:34:00.722+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:34:00.758+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.049 seconds
[2024-01-30T15:34:55.685+0100] {processor.py:161} INFO - Started process (PID=6895) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:34:55.687+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:34:55.688+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:34:55.687+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:34:55.692+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:34:55.691+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:34:55.693+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:34:55.722+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.041 seconds
[2024-01-30T15:35:44.910+0100] {processor.py:161} INFO - Started process (PID=6950) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:35:44.913+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:35:44.916+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:35:44.915+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:35:44.924+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:35:44.923+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:35:44.926+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:35:44.974+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.073 seconds
[2024-01-30T15:36:35.094+0100] {processor.py:161} INFO - Started process (PID=7001) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:36:35.096+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:36:35.098+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:36:35.097+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:36:35.102+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:36:35.101+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:36:35.102+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:36:35.131+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.040 seconds
[2024-01-30T15:37:28.979+0100] {processor.py:161} INFO - Started process (PID=7059) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:37:28.982+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:37:28.984+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:37:28.983+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:37:28.990+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:37:28.988+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:37:28.991+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:37:29.029+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.055 seconds
[2024-01-30T15:38:20.332+0100] {processor.py:161} INFO - Started process (PID=7119) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:38:20.334+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:38:20.335+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:38:20.335+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:38:20.339+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:38:20.339+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:38:20.340+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:38:20.371+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.042 seconds
[2024-01-30T15:39:19.966+0100] {processor.py:161} INFO - Started process (PID=7256) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:39:19.968+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:39:19.969+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:39:19.969+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:39:19.974+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:39:19.973+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:39:19.974+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:39:20.001+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.039 seconds
[2024-01-30T15:40:12.591+0100] {processor.py:161} INFO - Started process (PID=7314) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:40:12.596+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:40:12.599+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:40:12.598+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:40:12.608+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:40:12.606+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:40:12.608+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:40:12.730+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.147 seconds
[2024-01-30T15:40:43.341+0100] {processor.py:161} INFO - Started process (PID=7344) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:40:43.342+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:40:43.343+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:40:43.343+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:40:43.350+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:40:43.349+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:40:43.350+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:40:43.381+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.045 seconds
[2024-01-30T15:41:18.309+0100] {processor.py:161} INFO - Started process (PID=7374) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:41:18.312+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:41:18.314+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:41:18.313+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:41:18.321+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:41:18.318+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:41:18.321+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:41:18.350+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.047 seconds
[2024-01-30T15:42:20.419+0100] {processor.py:161} INFO - Started process (PID=7491) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:42:20.458+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:42:20.459+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:42:20.458+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:42:20.476+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:42:20.475+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:42:20.477+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:42:20.654+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.265 seconds
[2024-01-30T15:42:53.736+0100] {processor.py:161} INFO - Started process (PID=7661) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:42:53.738+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:42:53.741+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:42:53.741+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:42:53.747+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:42:53.746+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:42:53.748+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:42:53.793+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.063 seconds
[2024-01-30T15:43:50.003+0100] {processor.py:161} INFO - Started process (PID=7940) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:43:50.009+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:43:50.011+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:43:50.010+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:43:50.017+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:43:50.016+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:43:50.018+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:43:50.070+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.072 seconds
[2024-01-30T15:44:23.453+0100] {processor.py:161} INFO - Started process (PID=8066) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:44:23.456+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:44:23.457+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:44:23.457+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:44:23.467+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:44:23.463+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:44:23.467+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:44:23.499+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.049 seconds
[2024-01-30T15:44:58.353+0100] {processor.py:161} INFO - Started process (PID=8225) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:44:58.356+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:44:58.358+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:44:58.357+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:44:58.362+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:44:58.361+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:44:58.362+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:44:58.417+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.067 seconds
[2024-01-30T15:45:43.087+0100] {processor.py:161} INFO - Started process (PID=8277) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:45:43.090+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:45:43.091+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:45:43.090+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:45:43.095+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:45:43.094+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:45:43.095+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:45:43.122+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.038 seconds
[2024-01-30T15:46:23.778+0100] {processor.py:161} INFO - Started process (PID=8317) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:46:23.780+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:46:23.781+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:46:23.780+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:46:23.786+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:46:23.785+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:46:23.786+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:46:23.820+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.045 seconds
[2024-01-30T15:47:11.142+0100] {processor.py:161} INFO - Started process (PID=8367) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:47:11.146+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:47:11.149+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:47:11.148+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:47:11.155+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:47:11.153+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:47:11.155+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:47:11.189+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.052 seconds
[2024-01-30T15:47:44.690+0100] {processor.py:161} INFO - Started process (PID=8446) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:47:44.778+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:47:44.794+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:47:44.788+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:47:45.121+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:47:45.091+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:47:45.124+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:47:45.761+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 1.295 seconds
[2024-01-30T15:48:45.412+0100] {processor.py:161} INFO - Started process (PID=8679) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:48:45.433+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:48:45.443+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:48:45.438+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:48:45.489+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:48:45.484+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:48:45.489+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:48:45.822+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.424 seconds
[2024-01-30T15:51:43.176+0100] {processor.py:161} INFO - Started process (PID=9282) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:51:43.181+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:51:43.183+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:51:43.182+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:51:43.191+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:51:43.187+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:51:43.192+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:51:43.234+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.062 seconds
[2024-01-30T15:53:45.440+0100] {processor.py:161} INFO - Started process (PID=9730) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:53:45.471+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:53:45.502+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:53:45.476+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:53:45.586+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:53:45.511+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2024-01-30T15:53:45.590+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:53:46.059+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.677 seconds
[2024-01-30T15:55:17.098+0100] {processor.py:161} INFO - Started process (PID=9845) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:55:17.102+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:55:17.105+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:55:17.104+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:55:17.199+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:55:17.188+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from airflow.providers.apache.spark import SparkSubmitOperator
ImportError: cannot import name 'SparkSubmitOperator' from 'airflow.providers.apache.spark' (/home/ubuntu/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/__init__.py)
[2024-01-30T15:55:17.200+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:55:17.311+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.243 seconds
[2024-01-30T15:56:20.387+0100] {processor.py:161} INFO - Started process (PID=9910) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:56:20.393+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:56:20.396+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:56:20.395+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:56:20.467+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:56:20.460+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from program import ingest_data
  File "/home/ubuntu/airflow/dags/program.py", line 1, in <module>
    User
NameError: name 'User' is not defined
[2024-01-30T15:56:20.467+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:56:20.690+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.329 seconds
[2024-01-30T15:57:24.096+0100] {processor.py:161} INFO - Started process (PID=9970) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:57:24.121+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:57:24.135+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:57:24.126+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:57:24.217+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:57:24.212+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from program import ingest_data
  File "/home/ubuntu/airflow/dags/program.py", line 1, in <module>
    User
NameError: name 'User' is not defined
[2024-01-30T15:57:24.223+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:57:24.468+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.397 seconds
[2024-01-30T15:58:16.259+0100] {processor.py:161} INFO - Started process (PID=10028) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:58:16.299+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:58:16.306+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:58:16.306+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:58:16.356+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:58:16.354+0100] {dagbag.py:348} ERROR - Failed to import: /home/ubuntu/airflow/dags/FilmDag.py
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/FilmDag.py", line 6, in <module>
    from program import ingest_data
  File "/home/ubuntu/airflow/dags/program.py", line 1, in <module>
    User
NameError: name 'User' is not defined
[2024-01-30T15:58:16.361+0100] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:58:16.586+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.335 seconds
[2024-01-30T15:59:06.909+0100] {processor.py:161} INFO - Started process (PID=10072) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:59:06.916+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T15:59:06.923+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:59:06.922+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:59:06.969+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T15:59:07.590+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:59:07.588+0100] {override.py:1769} INFO - Created Permission View: can delete on DAG:MyDag
[2024-01-30T15:59:07.747+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:59:07.746+0100] {override.py:1769} INFO - Created Permission View: can edit on DAG:MyDag
[2024-01-30T15:59:07.850+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:59:07.850+0100] {override.py:1769} INFO - Created Permission View: can read on DAG:MyDag
[2024-01-30T15:59:07.859+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:59:07.859+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T15:59:07.895+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:59:07.895+0100] {dag.py:3055} INFO - Creating ORM DAG for MyDag
[2024-01-30T15:59:07.956+0100] {logging_mixin.py:188} INFO - [2024-01-30T15:59:07.955+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T15:59:08.406+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 1.505 seconds
[2024-01-30T16:00:00.905+0100] {processor.py:161} INFO - Started process (PID=10132) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:00:00.909+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:00:00.911+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:00:00.911+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:00:00.928+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:00:00.957+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:00:00.956+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:00:00.993+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:00:00.992+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T16:00:01.040+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.143 seconds
[2024-01-30T16:00:54.091+0100] {processor.py:161} INFO - Started process (PID=10189) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:00:54.097+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:00:54.099+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:00:54.098+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:00:54.171+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:00:54.246+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:00:54.245+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:00:54.294+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:00:54.294+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T16:00:54.336+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.261 seconds
[2024-01-30T16:01:51.773+0100] {processor.py:161} INFO - Started process (PID=10247) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:01:51.777+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:01:51.780+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:01:51.779+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:01:51.811+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:01:51.871+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:01:51.871+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:01:51.937+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:01:51.937+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T16:01:51.984+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.221 seconds
[2024-01-30T16:02:34.250+0100] {processor.py:161} INFO - Started process (PID=10297) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:02:34.254+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:02:34.264+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:02:34.264+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:02:34.303+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:02:34.404+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:02:34.403+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:02:34.590+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:02:34.588+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T16:02:34.761+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.516 seconds
[2024-01-30T16:03:25.459+0100] {processor.py:161} INFO - Started process (PID=10344) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:03:25.462+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:03:25.464+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:03:25.463+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:03:25.479+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:03:25.503+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:03:25.503+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:03:25.526+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:03:25.526+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T16:03:25.559+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.109 seconds
[2024-01-30T16:04:18.310+0100] {processor.py:161} INFO - Started process (PID=10404) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:04:18.314+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:04:18.316+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:04:18.315+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:04:18.356+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:04:18.515+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:04:18.514+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:04:18.560+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:04:18.560+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T16:04:18.658+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.355 seconds
[2024-01-30T16:05:05.632+0100] {processor.py:161} INFO - Started process (PID=10448) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:05:05.635+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:05:05.636+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:05:05.636+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:05:05.650+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:05:05.671+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:05:05.671+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:05:05.709+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:05:05.709+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T16:05:05.864+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.271 seconds
[2024-01-30T16:05:56.926+0100] {processor.py:161} INFO - Started process (PID=10486) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:05:56.939+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:05:56.940+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:05:56.940+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:05:56.987+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:05:57.082+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:05:57.082+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:05:57.185+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:05:57.185+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-01T00:00:00+00:00, run_after=2022-01-02T00:00:00+00:00
[2024-01-30T16:05:57.416+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.524 seconds
[2024-01-30T16:11:40.719+0100] {processor.py:161} INFO - Started process (PID=10672) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:11:40.728+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T16:11:40.731+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:11:40.730+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:11:40.832+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T16:11:41.407+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:11:41.406+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T16:11:41.486+0100] {logging_mixin.py:188} INFO - [2024-01-30T16:11:41.486+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2022-01-02T00:00:00+00:00, run_after=2022-01-03T00:00:00+00:00
[2024-01-30T16:11:42.001+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 1.283 seconds
[2024-01-30T22:42:22.361+0100] {processor.py:161} INFO - Started process (PID=4241) to work on /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T22:42:22.376+0100] {processor.py:830} INFO - Processing file /home/ubuntu/airflow/dags/FilmDag.py for tasks to queue
[2024-01-30T22:42:22.384+0100] {logging_mixin.py:188} INFO - [2024-01-30T22:42:22.380+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T22:42:22.530+0100] {processor.py:840} INFO - DAG(s) 'MyDag' retrieved from /home/ubuntu/airflow/dags/FilmDag.py
[2024-01-30T22:42:22.774+0100] {logging_mixin.py:188} INFO - [2024-01-30T22:42:22.773+0100] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-30T22:42:22.898+0100] {logging_mixin.py:188} INFO - [2024-01-30T22:42:22.898+0100] {dag.py:3820} INFO - Setting next_dagrun for MyDag to 2024-01-01T00:00:00+00:00, run_after=2024-01-02T00:00:00+00:00
[2024-01-30T22:42:23.107+0100] {processor.py:183} INFO - Processing /home/ubuntu/airflow/dags/FilmDag.py took 0.797 seconds
